src/components/Dropdown/Dropdown.tsx

```diff
@@ -252,9 +265,11 @@ const Dropdown = ({
   color,
   elevation = 0,
   multi = false,
-  name,
+  name = 'dropdown',
```

Comment:
Is this something we should generate a unique value for? What are we getting by defaulting this here? 

and if the dev does not specify the name attribute do we run into unintended issues if there are multiple dropdowns  with the same name on the page?

Reply:
https://github.com/Headstorm/foundry-ui/pull/241#discussion_r678444878